rosidl_generator_py/resource/_msg.py.em

```diff
@@ -376,7 +376,7 @@ if member.name in dict(inspect.getmembers(builtins)).keys():
             from collections.abc import Set
             from collections import UserList
             from collections import UserString
-@[  elif isinstance(type_, AbstractString) and type_.has_maximum_size()]@
+@[  elif isinstance(type_, AbstractGenericString) and type_.has_maximum_size()]@
```

Comment:
What about the condition at L394 too?

Reply:
Very good catch. Yes, that condition definitely needs update to check the length of WStrings too: 7fc6f0d2896faf8dba62d1edce3d0244ca2c115c